website/source/docs/configuration/interpolation.html.md

```diff
@@ -80,6 +80,31 @@ The supported built-in functions are:
   * `base64encode(string)` - Returns a base64-encoded representation of the
     given string.
 
+  * `cidrfirstaddr(iprange)` - Takes an IP address range in CIDR notation
+    and returns the first IP address in that range. This is the same
+    as simply removing the prefix indicator from the end. For example,
+    ``cidrfirstaddr("10.0.0.0/8", 2)`` returns ``10.0.0.0``.
```

Comment:
The behavior of this doesn't match what I'm used to seeing in CIDR manipulation tooling. Here's the output of `ipcalc` on that example CIDR:

```
❯ ipcalc 10.0.0.0/8
Address:   10.0.0.0             00001010. 00000000.00000000.00000000
Netmask:   255.0.0.0 = 8        11111111. 00000000.00000000.00000000
Wildcard:  0.255.255.255        00000000. 11111111.11111111.11111111
=>
Network:   10.0.0.0/8           00001010. 00000000.00000000.00000000
HostMin:   10.0.0.1             00001010. 00000000.00000000.00000001
HostMax:   10.255.255.254       00001010. 11111111.11111111.11111110
Broadcast: 10.255.255.255       00001010. 11111111.11111111.11111111
```

I'd expect `cidrfirstaddr` to match `HostMin` there, but it does not. Is there a use case I'm missing?